relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -139,28 +139,15 @@ impl NormalizeVisitor {
         Self::collapse_items(&mut collapse, &mut select.projection);
     }
 
-    fn transform_from(&mut self, from: &mut [TableWithJoins]) {
-        // Iterate "FROM".
-        for from in from {
-            self.transform_table_factor(&mut from.relation);
-            for join in &mut from.joins {
-                self.transform_table_factor(&mut join.relation);
+    fn simplify_table_alias(alias: &mut Option<TableAlias>) {
+        if let Some(TableAlias { name, columns }) = alias {
+            Self::scrub_name(name);
+            for column in columns {
+                Self::scrub_name(column);
             }
         }
     }
 
-    fn transform_table_factor(&mut self, table_factor: &mut TableFactor) {
-        match table_factor {
-            // Recurse into subqueries.
-            TableFactor::Derived { subquery, .. } => {
-                self.transform_query(subquery);
-            }
-            // Strip quotes from identifiers in table names.
-            TableFactor::Table { name, .. } => Self::simplify_compound_identifier(&mut name.0),
-            _ => {}
-        }
-    }
```

Comment:
Manual recursion is not necessary here. `sqlparser` calls `pre_visit_table_factor` for every table factor.